webapp/src/components/App/Singlestudy/explore/Configuration/TimeSeriesManagement/utils.ts

```diff
@@ -12,76 +12,19 @@
  * This file is part of the Antares project.
  */
 
-import type { DeepPartial } from "react-hook-form";
-import type { StudyMetadata } from "../../../../../../common/types";
-import client from "../../../../../../services/api/client";
+import { TSType } from "@/services/api/studies/timeseries/constants";
+import type { TSTypeConfig, TSConfigDTO, TTSType } from "@/services/api/studies/timeseries/types";
 
-////////////////////////////////////////////////////////////////
-// Enums
-////////////////////////////////////////////////////////////////
+export type TSConfigValues = Record<TTSType, TSTypeConfig & { enable: boolean }>;
 
-export enum TSType {
-  Load = "load",
-  Hydro = "hydro",
-  Thermal = "thermal",
-  Wind = "wind",
-  Solar = "solar",
-  Renewables = "renewables",
-  NTC = "ntc",
-}
-
-enum SeasonCorrelation {
-  Monthly = "monthly",
-  Annual = "annual",
-}
-
-////////////////////////////////////////////////////////////////
-// Types
-////////////////////////////////////////////////////////////////
-
-interface TSFormFieldsForType {
-  stochasticTsStatus: boolean;
-  number: number;
-  refresh: boolean;
-  refreshInterval: number;
-  seasonCorrelation: SeasonCorrelation;
-  storeInInput: boolean;
-  storeInOutput: boolean;
-  intraModal: boolean;
-  interModal: boolean;
-}
-
-export interface TSFormFields
-  extends Record<
-    Exclude<TSType, TSType.Thermal | TSType.Renewables | TSType.NTC>,
-    TSFormFieldsForType
-  > {
-  [TSType.Thermal]: Omit<TSFormFieldsForType, "seasonCorrelation">;
-  [TSType.Renewables]: Pick<
-    TSFormFieldsForType,
-    "stochasticTsStatus" | "intraModal" | "interModal"
-  >;
-  [TSType.NTC]: Pick<TSFormFieldsForType, "stochasticTsStatus" | "intraModal">;
-}
-
-////////////////////////////////////////////////////////////////
-// Constants
-////////////////////////////////////////////////////////////////
-
-export const DEFAULT_VALUES: DeepPartial<TSFormFields> = {
-  thermal: {
-    stochasticTsStatus: false,
-    number: 1,
-  },
-};
-
-////////////////////////////////////////////////////////////////
-// Functions
-////////////////////////////////////////////////////////////////
+export const defaultValues = Object.values(TSType).reduce((acc, type) => {
```

Comment:
I don't see anything wrong with putting it in the component file at top. but as you prefer